app/src/filesystem_desktop_linux.cc

```diff
@@ -55,7 +55,7 @@ std::string HomeDir(std::string* out_error) {
 std::string XdgDataHomeDir(std::string* out_error) {
   const char* data_home = getenv("XDG_DATA_HOME");
   if (data_home) return data_home;
-  return "";
+  return "/data";
```

Comment:
Can be ignored, was just testing if this fixed another issue, get bad_alloc with or without this